src/main/java/TaskListener.java

```diff
@@ -0,0 +1,191 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class TaskListener {
+    static final String DIVIDER = "--------------------------------------------------------------------";
+    private TaskList taskList;
+
+    public TaskListener(int expectedTasksCount) {
+        this.taskList = new TaskList(expectedTasksCount);
+    }
+
+    private static void printLines(String... lines) {
+        System.out.println(DIVIDER);
+        for (String line : lines) {
+            System.out.println(line);
+        }
+        System.out.println(DIVIDER + System.lineSeparator());
+    }
+
+    private void printAddedTask(Task task) {
+        String firstSentence = "Got it. I've added this task:";
+        String tasksWord = taskList.getTasksCount() == 1 ? " task" : " tasks";
+        String secondSentence = "Now you have " + taskList.getTasksCount() + tasksWord + " in the list.";
+        printLines(firstSentence + System.lineSeparator() + task + System.lineSeparator() + secondSentence);
+    }
+
+    public static void greet() {
+        String logo =
+                        "   _____ __  ____________  __    ____  ________ __\n" +
+                        "  / ___// / / / ____/ __ \\/ /   / __ \\/ ____/ //_/\n" +
+                        "  \\__ \\/ /_/ / __/ / /_/ / /   / / / / /   / ,<   \n" +
+                        " ___/ / __  / /___/ _, _/ /___/ /_/ / /___/ /| |  \n" +
+                        "/____/_/ /_/_____/_/ |_/_____/\\____/\\____/_/ |_|  ";
+
+
+        printLines("Hello from", logo);
+
+        printLines("Hello! I'm SHERLOCK", "What can I do for you?");
+    }
+
+    private void addCommandHandler(String name) {
+        this.taskList.addTask(new Task(name, false));
+        printLines("added: " + name);
+    }
+
+    private void todoCommandHandler(String arguments) {
```

Comment:
method name is "noun". it might be better to use a "verb" such as handleTodoTask